paparazzi/src/main/java/app/cash/paparazzi/internal/resources/ProjectResourceRepository.kt

```diff
@@ -19,20 +19,39 @@ internal class ProjectResourceRepository private constructor(
 
   companion object {
     fun create(
-      resourceDirectories: List<File>
+      resourceDirectories: List<File>,
+      moduleResourceDirectories: List<File>
     ): ProjectResourceRepository {
       return ProjectResourceRepository(
         displayName = "main",
-        // TODO: need mapOf(package to listOf(resourceDirectory)) for each transitive project module
-        localResources = listOf(
-          ModuleResourceRepository.forMainResources(
-            namespace = getNamespace(namespacing = ResourceNamespacing.DISABLED, packageName = "TODO"),
-            resourceDirectories = resourceDirectories
-          )
-        )
+        localResources = computeRepositories(resourceDirectories, moduleResourceDirectories)
       )
     }
 
+    private fun computeRepositories(
+      resourceDirectories: List<File>,
+      moduleResourceDirectories: List<File>
+    ): List<LocalResourceRepository> {
+      val main: LocalResourceRepository = getModuleResources(resourceDirectories)
+
+      val resources: MutableList<LocalResourceRepository> = ArrayList(moduleResourceDirectories.size + 1)
+      resources += main
+      for (moduleResourceDirectory in moduleResourceDirectories) {
+        resources += getModuleResources(listOf(moduleResourceDirectory))
+      }
+
+      return resources
```

Comment:
```suggestion
      val resources = buildList< LocalResourceRepository>(moduleResourceDirectories.size + 1) {
        this += main
        for (moduleResourceDirectory in moduleResourceDirectories) {
          this += getModuleResources(listOf(moduleResourceDirectory))
        }
      }

      return resources
```

Reply:
can the for loop be simplified to:

```
this += getModuleResources(moduleResourceDirectories)
```
?

Reply:
> can the for loop be simplified to:
> 
> ```
> this += getModuleResources(moduleResourceDirectories)
> ```
> 
> ?

As discussed, `getModuleResources(moduleResourceDirectories)` will reverse the order again. Each `moduleResourceDirectory` should be in its own list and keep original order